csharp/src/Ice/PluginInitializationContext.cs

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+using System;
+
+namespace ZeroC.Ice
+{
+    public sealed class PluginInitializationContext : IDisposable
```

Comment:
Could we use a ref struct for `PluginInitializationContext` to ensure the plugin can't keep it around? (Then no need for the dispose mechanism).

Reply:
Yes fixed, that is much simpler now